src/test/java/com/iota/iri/conf/ConfigTest.java

```diff
@@ -303,7 +299,7 @@ public void backwardsIniCompatibilityTest() {
                 .map(Enum::name)
                 // make it explicit that we have removed some configs
                 .filter(config -> !ArrayUtils.contains(new String[]{"CONFIG", "TESTNET", "DEBUG",
-                        "MIN_RANDOM_WALKS", "MAX_RANDOM_WALKS"}, config))
+                        "MIN_RANDOM_WALKS", "MAX_RANDOM_WALKS", "MAX_PEERS"}, config))
```

Comment:
Should `UDP_RECEIVER_PORT` and `TCP_RECEIVER_PORT` be added here?

Reply:
:+1: